Source/Fuse.Reactive.Expressions/MathFunctions.uno

```diff
@@ -33,4 +34,376 @@ namespace Fuse.Reactive
 			return "max(" + Left + ", " + Right + ")";
 		}
 	}
+	
+	[UXFunction("mod")]
+	public sealed class Mod : BinaryOperator
+	{
+		[UXConstructor]
+		public Mod([UXParameter("Left")] Expression left, [UXParameter("Right")] Expression right): base(left, right) {}
+		protected override object Compute(object left, object right)
+		{
+			return Math.Mod( Marshal.ToFloat(left), Marshal.ToFloat(right) );
+		}
+
+		public override string ToString()
+		{
+			return "mod(" + Left + ", " + Right + ")";
+		}
+	}
+
+	[UXFunction("even")]
+	/** True if the rounded value is even, false otherwise*/
+	public sealed class Even : UnaryOperator
+	{
+		[UXConstructor]
+		public Even([UXParameter("Operand")] Expression operand): base(operand) {}
+		protected override object Compute(object operand)
+		{
+			//this rounds floats automatically it seems
+			var q = (int)Math.Floor(Marshal.ToType<float>(operand)+0.5f);
```

Comment:
Yes it is. It does round-to-positive-infinity on 0.5 values. Math.Round does this as well, but seeing as though it's the same I'll just use Math.Round instead.